packages/server/src/fhir/operations/conceptmaptranslate.ts

```diff
@@ -0,0 +1,182 @@
+import { parseInputParameters, sendOutputParameters } from './utils/parameters';
+import { Operator, allOk, badRequest, notFound } from '@medplum/core';
+import { Request, Response } from 'express';
+import { getOperationDefinition } from './definitions';
+import { CodeableConcept, Coding, ConceptMap, ConceptMapGroup, OperationOutcome } from '@medplum/fhirtypes';
+import { getAuthenticatedContext } from '../../context';
+import { sendOutcome } from '../outcomes';
+
+const operation = getOperationDefinition('ConceptMap', 'translate');
+
+type ConceptMapTranslateParameters = {
+  url?: string;
+  code?: string;
+  system?: string;
+  source?: string;
+  coding?: Coding;
+  codeableConcept?: CodeableConcept;
+  targetsystem?: string;
+};
+
+type Match = {
+  equivalence?: string;
+  concept?: Coding;
+};
+
+type ConceptMapTranslateOutput = {
+  result: boolean;
+  message?: string;
+  match?: Match[];
+};
+
+export async function conceptMapTranslateHandler(req: Request, res: Response): Promise<void> {
+  const params = parseInputParameters<ConceptMapTranslateParameters>(operation, req);
+
+  const map = await lookupConceptMap(params, req.params.id);
+  if (isOutcome(map)) {
+    sendOutcome(res, map);
+    return;
+  } else if (!map.group) {
```

Comment:
Another super minor thing, just for aligning coding styles, is using `else` when the parent block uses `return`  Functionally, totally equivalent, so this is purely a style question.  It'd just be nice to be consistent throughout the code base.

Reply:
I generally favor an early-return style for error cases, leaving the main code path unindented to highlight the "happy path" when reading the code — hence no `else` here.  Thoughts on that general idea?

Reply:
Ah, I see — I do like to group error cases into an `if`/`else if` as well, to visually cluster them together and further highlight the happy path.  If you'd prefer it the other way, we could enforce that style with https://eslint.org/docs/latest/rules/no-else-return by setting `allowElseIf: false`